File: Linguagem C++/Estrutura Enquanto/SenhaFixa.cpp
#include <iostream>

using namespace std;

int main()
{
    int senha;


    cout << "Digite a senha: ";
    cin >> senha;

    while (senha != 2002){
        cout << "Senha invalida. Tente Novamente: ";
        cin >> senha;
    };

    cout << "Acesso permitido." << endl;
    return 0;
}

File: Linguagem Java/Exercicios/SenhaFixa.java
import java.util.Locale;
import java.util.Scanner;

public class SenhaFixa {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int senha;
		
		System.out.print("Digite a senha: ");
        senha = sc.nextInt();
        
        while (senha != 2002) {
        	System.out.print("Senha invalida.. Tente novamente: ");
        	senha = sc.nextInt();
        };
        
        System.out.println("Acesso permitido !");
        
        sc.close();
	}

}
